group_vars/all/openshift_routes.yml

```diff
@@ -1,16 +1,14 @@
 ---
-# URL
+# Url
```

Comment:
URL is an acronym. I think it must stay in upercasse.

Reply:
url is always mentioned in low cap in code ;-)

Reply:
url is always mentioned in low cap in code ;-)